rs/replicated_state/src/canister_snapshots.rs

```diff
@@ -27,10 +27,14 @@ pub struct CanisterSnapshots {
 }
 
 impl CanisterSnapshots {
-    pub fn new(
-        snapshots: BTreeMap<SnapshotId, Arc<CanisterSnapshot>>,
-        snapshot_ids: BTreeMap<CanisterId, BTreeSet<SnapshotId>>,
-    ) -> Self {
+    pub fn new(snapshots: BTreeMap<SnapshotId, Arc<CanisterSnapshot>>) -> Self {
```

Comment:
Is this function ever called (at least in tests)?

Reply:
It is called in the test I just added. It will be called as part of loading checkpoints from disk (but that code is still on the way).